Replace mentions of 'oAuth', by more accurate 'OAuth'. (See https://oauth.net/) (#745)
codenvy/che-docs: src/main/pages/che-6/setup-docker/docker-single-user.adoc

[id="configuration"]
== Configuration

> Che CLI allows a wide range of configuration changes to set up the port, hostname, OAuth, Docker, git, and to solve networking issues. For details, see link:docker-config.html[Che configuration on Docker].

Reviewer: ```suggestion
Che CLI supports configuring the port, hostname, OAuth, Docker, Git, and resolving networking issues. For details, see link:docker-config.html[Che configuration on Docker].
```
